content/articles/inbo_stats_courses/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Statistics courses @ INBO!"
+description: "Links to course material of statistics courses taught at INBO (only accessible for INBO employees)"
+date: 2021-02-24
+categories: ["r", "statistics"]
+tags: ["r", "INLA", "mixed model", "uncertainty", "explorative data analysis", "analysis of variance", "generalized linear regression", "multivariate statistics"]
+output: 
+    md_document:
+        preserve_yaml: true
+        variant: gfm
+---
+
+During the course of the years, we have taught many statistics courses at the Research Institute for Nature and Forest (INBO), both for scientists and technicians.
+Most of these courses have been developed and taught in-house.
+Some (mostly advanced) courses were taught in-house by external teachers.
```

Comment:
```suggestion
Some (mostly advanced) courses were taught in-house by external teachers.
```
I'm not sure if we should mention external courses here as well...

Reply:
I think that is OK. I will leave it as is.